doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+.. _plutus_tools_component_descriptions:
+
```

Comment:
I don't see the playground should it be in here? Deprecated!

Reply:
You want to list the playground and mark it as deprecated?

Reply:
I think it would be better to not mention it at all if it is truly deprecated. Other thoughts?